src/player.ts

```diff
@@ -351,6 +351,10 @@ export default class Player extends EventEmitter {
     return this.data.gaiaPowerTokens() >= 4 && this.faction === Faction.Itars && this.data.hasPlanetaryInstitute();
   }
 
+  canUpgradeResearchBaltaks( field: ResearchField): boolean {
+    return (this.data.hasPlanetaryInstitute() && field === ResearchField.Navigation) || field !== ResearchField.Navigation;
```

Comment:
no need for parentheses around the return

Reply:
ok